src/pages/map.tsx

```diff
@@ -49,7 +52,14 @@ const Map = ReactMapboxGl({
   accessToken: process.env.NEXT_PUBLIC_MAPBOX_KEY
 });
 
-const Mark = ({ lokasi }) => (
+type LocationData = Partial<DetailLokasi> & {
+  jadwal?: Jadwal[] | null;
+};
```

Comment:
use interface 

```suggestion
interface LocationData extends Partial<DetailLokasi> {
  jadwal?: Jadwal[] | null;
};
```

Reply:
Added  in https://github.com/tibudiyanto/jakarta-vax-availability/pull/34/commits/6259a467af374b737231680a0d41c5ce906323d4